public/assets/js/scripts.js

```diff
@@ -1,6 +1,6 @@
 // show questions based on slider on faq page
 function toggleQuestions(id) {
-  $('#payment, #profile, #security, #getting-started').hide();
+  $("#payment, #profile, #security, #getting-started").hide();
```

Comment:
Can we use single quotes for strings instead of double quotes? Please see https://airbnb.io/javascript/#strings